### PR TITLE
[CORE] Upgrade Custom Elements to 1.0.0

### PR DESCRIPTION
**Pull request by adityarb88**
Fixes: #916 

1. Update Custom Elements
2. Get rid of Mutation Observer (because we wont support IE10 from the next release)

Tests: IE11, Edge, Chrome, Safari, Firefox

Additional tests for firefox and IE11 to check if this issue does not reoccur: #546 

Prepublished and tested on the website.

Signed-off-by: Aditya Bhandari <adityab@vmware.com>

### CODE REVIEW

File: build/npm/clarity-icons-README.md [hunk 1]
```diff
@@ -7,19 +7,13 @@
...
+    npm install @webcomponents/custom-elements@1.0.0
     ```
 
-3. If your application supports IE10, the polyfill will require the MutationObserver shim to work. If your application doesn't support IE10, you can skip the following installation:
-    ```
-    npm install mutationobserver-shim@0.3.2
-    ```
-
-4. Include the clarity-icons.min.css and clarity-icons.min.js in your HTML. As custom-elements.min.js is dependent on the Custom Elements polyfill, make sure to include it before clarity-icons.min.js. Also, if your app needs to support IE10, include the mutationobserver.min.js before the polyfill:
+3. Include the clarity-icons.min.css and clarity-icons.min.js in your HTML. As custom-elements.min.js is dependent on the Custom Elements polyfill, make sure to include it before clarity-icons.min.js:
```
Comment (Shijir) [inline build/npm/clarity-icons-README.md]: >  As custom-elements.min.js is dependent on the Custom Elements polyfill,

You have custom-elements.min.js is dependent on  the Custom Elements polyfill.
It should be Clarity Icons is dependent on the Custom Elements polyfill, right?

Comment (adityarb88) [inline build/npm/clarity-icons-README.md]: Yep. will fix that.

Comment (adityarb88) [inline build/npm/clarity-icons-README.md]: done

File: README.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ If you already have an Angular application, you can follow the installation step
 
 2. Install the polyfill for Custom Elements: 
     ```bash
-    npm install @webcomponents/custom-elements@1.0.0-rc.3 --save
+    npm install @webcomponents/custom-elements@1.0.0 --save
     ```
 
 3. (Optional) If your application supports IE10, the polyfill will require the MutationObserver shim to work. If your
```
Comment (Shijir) [inline README.md]: @adityarb88 Do we need this step? Because users now don't need to install the MutationObserver shim, right?

Comment (adityarb88) [inline README.md]: fixed. i thought i updated it but i guess i didn't :-/

**Pull request merged**